combine-plants.lua

```diff
@@ -16,32 +16,38 @@ if args.max then max = tonumber(args.max) end
 
 local stockpile = nil
 if args.stockpile then stockpile = df.building.find(tonumber(args.stockpile)) end
-
+    
```

Comment:
Should get rid of this (and on new lines 47 and 78)
(We should probably get travis set up in this repo eventually.)